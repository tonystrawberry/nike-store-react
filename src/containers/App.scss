.app__container-loading{
  padding: 30px;
  height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
}