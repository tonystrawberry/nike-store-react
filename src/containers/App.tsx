import React from 'react';
import './App.css';
import Header from '../components/Header';
import ProductOverviewContainer from './ProductOverviewContainer'

function App() {
  return (
    <div>
      <Header />
      <ProductOverviewContainer />
    </div>
  );
}

export default App;
