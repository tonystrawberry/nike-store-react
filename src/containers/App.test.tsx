import { shallow, mount } from 'enzyme';
import React from 'react';
import App from './App';
import { MemoryRouter } from 'react-router-dom';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';

let store;
var initialState = {
  selectProduct: {
    selected: false,
    selectedProductId: null,
    products: [],
    loading: false
  }
}
const mockStore = configureStore([]);

it ('expect to render App component on /', () => {
  store = mockStore(initialState);
  const wrapper = mount(
    <MemoryRouter initialEntries={["/"]}>
      <Provider store={store}><App /></Provider>
    </MemoryRouter>
    )
  expect(wrapper.find('header'));
})