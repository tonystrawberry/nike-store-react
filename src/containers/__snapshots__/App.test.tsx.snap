// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render App component 1`] = `
<div>
  <Header />
  <Connect(ProductOverviewContainer) />
</div>
`;
